composer.json

```diff
@@ -27,20 +27,19 @@
         "sort-packages": true,
         "allow-plugins": {
             "dealerdirect/phpcodesniffer-composer-installer": true,
-            "composer/package-versions-deprecated": true,
-            "laminas/laminas-dependency-plugin": true
+            "composer/package-versions-deprecated": true
         }
     },
     "require": {
         "php": "~7.4.0 || ~8.0.0 || ~8.1.0",
+        "composer-plugin-api": "^2.0",
```

Comment:
Why is this here?

Reply:
I need to know the version of the installed `psr/container` version.
Previously, we used `ocramius/package-versions` but since composer adds that feature as of composer v2, we should require users to use the composer v2 to avoid an additional dependency (such as `composer/package-versions-deprecated`).

Reply:
As decided in the latest TSC meeting: we avoid the usage of `composer/package-versions-deprecated` and instead use `composer-plugin-api` with at least v2.